src/importexport/musicxml/internal/musicxml/exportxml.cpp

```diff
@@ -3296,28 +3296,51 @@ static void writeChordLines(const Chord* const chord, XmlWriter& xml, Notations&
 static void writeBreathMark(const Breath* const breath, XmlWriter& xml, Notations& notations, Articulations& articulations)
 {
     if (breath && ExportMusicXml::canWrite(breath)) {
+        QString tagName;
+        QString type;
+
         notations.tag(xml, breath);
         articulations.tag(xml);
         if (breath->isCaesura()) {
-            xml.tag("caesura");
+            tagName = "caesura";
+            switch (breath->symId()) {
+            case SymId::caesuraCurved:
+                type = "curved";
+                break;
+            case SymId::caesuraShort:
+                type = "short";
+                break;
+            case SymId::caesuraThick:
+                type = "thick";
+                break;
+            case SymId::caesuraSingleStroke:
+            case SymId::chantCaesura:
+                type = "single";
+                break;
+            case SymId::caesura:
+            default:
+                type = "normal";
+            }
         } else {
-            QString breathMarkType;
+            tagName = "breath-mark";
             switch (breath->symId()) {
             case SymId::breathMarkTick:
-                breathMarkType = "tick";
+                type = "tick";
                 break;
             case SymId::breathMarkUpbow:
-                breathMarkType = "upbow";
+                type = "upbow";
                 break;
             case SymId::breathMarkSalzedo:
-                breathMarkType = "salzedo";
+                type = "salzedo";
                 break;
             default:
-                breathMarkType = "comma";
+                type = "comma";
             }
-
-            xml.tag("breath-mark", breathMarkType);
         }
+        tagName += color2xml(breath);
+        tagName += QString(" placement=\"%1\"").arg(breath->placement() == PlacementV::ABOVE ? "above" : "below");
+
+        xml.tagRaw(tagName, type);
```

Comment:
Unfortunately MuseScore 3.6 and older can't import those caesuras, but report a corrupt score instead, on every measure after a caesura like this:
```xml
        <notations>
          <articulations>
            <caesura>curved</caesura>
            </articulations>
          </notations>
 ```
It only seems to accept the old
```xml
        <notations>
          <articulations>
            <caesura/>
            </articulations>
          </notations>
 ```
Ideas welcome!

Reply:
My 3.7 will of course fix this ;-)